src/sentry_plugins/slack/plugin.py

```diff
@@ -152,18 +166,18 @@ def notify(self, notification):
 
         # They can be the same if there is no culprit
         # So we set culprit to an empty string instead of duplicating the text
-        if culprit and title != culprit:
+        if self.get_option('include_culprit', project) and culprit and title != culprit:
```

Comment:
The default only gets applied in the configuration form. The code that calls `get_options` is in the main sentry repo: https://github.com/getsentry/sentry/blob/master/src/sentry/plugins/base/v1.py#L120 which in turn calls https://github.com/getsentry/sentry/blob/master/src/sentry/plugins/helpers.py#L38

I agree it'd probably be simplest to change this to `exclude_project` and `exclude_culprit` and they can default to False.

Thanks!